matminer/featurizers/composition_deprecated.py

```diff
@@ -1,3 +1,5 @@
+## Deprecated 
```

Comment:
I'd say just remove this file, ti's confusing to have a specific composition deprecated file outside of the composition subpackage

Reply:
Unless this is being used in some way I'm not familiar with, like what is the purpose of this file now?

Reply:
I was using it to check that everything was still working in the same way. I will remove it once the PR is ready but I found it helpful to keep it for reference purposes in the meantime